test/avdl/sample.avdl

```diff
@@ -57,7 +57,7 @@ protocol sample {
 		case BOZO: void;
 		case BIPPY: int;
 		case AGGLE: void;
-		case FLAGGLE: void;
+		case FLAGGLE: EnumNoString;
```

Comment:
does this mean we lose test coverage and no longer had the `case FOO: void` case covered?